.gitignore

```diff
@@ -74,3 +74,9 @@ spark-warehouse
 
 tikv-client/src/main/java/com/pingcap/tikv/parser/*.interp
 tikv-client/src/main/java/com/pingcap/tikv/parser/*.tokens
+tikv-client/src/main/java/com/pingcap/tikv/parser/MySqlLexer.java
```

Comment:
Put those into generated source folder and ignore like proto files.

Reply:
DONE